docs/releases/1.9.txt

```diff
@@ -356,6 +356,15 @@ Backwards incompatible changes in 1.9
     deprecation timeline for a given feature, its removal may appear as a
     backwards incompatible change.
 
+Paginator.page_range
+~~~~~~~~~~~~~~~~~~~~
+An inconsistency in the behavior of ``Paginator.page_range`` across major
+versions of Python has been resolved.
+
+Previously, ``Paginator.page_range`` would return a ``list`` in Python 2.x,
```

Comment:
It would also be helpful to note what user code might be affected and how to update it (add `list()` I guess).